src/cli/podcast_commands.py

```
@@ -362,6 +364,78 @@ def cleanup_audio(args, config: Config):
+                print(f"  {stage}: {count}")
+            print()
+
+            # Run workflow
+            if args.stage:
+                logger.info(f"Running stage: {args.stage}")
+                result = orchestrator.run_stage(args.stage)
+                print(f"\nStage '{args.stage}' complete:")
+                print(f"  Processed: {result.processed}")
+                print(f"  Failed: {result.failed}")
+                if result.errors:
+                    print(f"  Errors:")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)